src/main/java/org/pac4j/dropwizard/Pac4jBundle.java

```diff
@@ -18,33 +26,51 @@
  * Dropwizard application's Jersey filter chain.
  */
 public abstract class Pac4jBundle<T extends Configuration>
-        implements ConfiguredBundle<T>, Pac4jConfiguration<T> {
+        implements ConfiguredBundle<T> {
     @Override
     public final void initialize(Bootstrap<?> bootstrap) {
-        // nothing to do
+        bootstrap.getObjectMapper().addMixIn(Client.class,
+                Pac4JMixins.ClientMixin.class);
+        bootstrap.getObjectMapper().addMixIn(BaseClient.class,
+                Pac4JMixins.BaseClientMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Authenticator.class,
+                Pac4JMixins.AuthenticatorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(PasswordEncoder.class,
+                Pac4JMixins.PasswordEncoderMixin.class);
+        bootstrap.getObjectMapper().addMixIn(CredentialsExtractor.class,
+                Pac4JMixins.CredentialsExtractorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(ProfileCreator.class,
+                Pac4JMixins.ProfileCreatorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(AuthorizationGenerator.class,
+                Pac4JMixins.AuthorizationGeneratorMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Authorizer.class,
+                Pac4JMixins.AuthorizerMixin.class);
+        bootstrap.getObjectMapper().addMixIn(Matcher.class,
+                Pac4JMixins.MatcherMixin.class);
```

Comment:
You are entirely right, it's missing, and it's not a problem at all to add it. Actually, I had it noted on a piece of paper but forgot about it ^^.
I would like to make the use case with a database work smoothly and be wholly configurable in the YAML file if possible. For now it isn't, mainly because of the interaction between the database as declared in the YAML and it's use in pac4j's configuration, also in the YAML file. This question is related to the one discussed with @evnm  just above: currently to fully setup one of the database authenticator, you need to do it outside of the config file.